src/messages.ts

```diff
@@ -27,6 +27,12 @@ export const messages = {
   projects: {
     title: 'پروژه ها',
     create: 'ایجاد پروژه جدید',
+    notification: {
+      successfulCreate: (name: string) =>
```

Comment:
Please use the message from the design.
we should keep the design in sync with implementation as much as possible.
if you prefer this, share it with me and I'll change the design